src/java.base/share/classes/java/lang/Math.java

```diff
@@ -1527,7 +1527,7 @@ public static long absExact(long a) {
      */
     @IntrinsicCandidate
     public static float abs(float a) {
-        return (a <= 0.0F) ? 0.0F - a : a;
+        return Float.intBitsToFloat(Float.floatToRawIntBits(a) & 0x7fffffff);
```

Comment:
I'd like to see a brief comment on these methods along the lines of "zero out sign bit in the representation." Using underscores in the literal to better group the digits or using the constants in jdk.internal.math.DoubleConsts.

Reply:
There is no specific constant in `{Float,Double}Consts` to mask all but the sign bit, but I had thought it might be good to add one there.